mteb/tasks/Classification/multilingual/LanguageClassification.py

```diff
@@ -5,26 +5,26 @@
 from ....abstasks import AbsTaskClassification
 
 _LANGUAGES = {
-    "ar": ["ara-Arab"],
-    "bg": ["bul-Cyrl"],
-    "de": ["deu-Latn"],
-    "el": ["ell-Grek"],
-    "en": ["eng-Latn"],
-    "es": ["spa-Latn"],
-    "fr": ["fra-Latn"],
-    "hi": ["hin-Deva"],
-    "it": ["ita-Latn"],
-    "ja": ["jpn-Jpan"],
-    "nl": ["nld-Latn"],
-    "pl": ["pol-Latn"],
-    "pt": ["por-Latn"],
-    "ru": ["rus-Cyrl"],
-    "sw": ["swa-Latn"],
-    "th": ["tha-Thai"],
-    "tr": ["tur-Latn"],
-    "ur": ["urd-Arab"],
-    "vi": ["vie-Latn"],
-    "zh": ["cmn-Hans"],
+    "ara": ["ara-Arab"],
```

Comment:
This is still an incorrect format. "ara" or "ar" refer to the subset on huggingface. However there is no subset on HF only the "default". Thus _LANGUAGES should either be a list [LANG1, LANG2,...] or a dict {"default": [LANG1, LANG2, ...}